kf_lib_data_ingest/etl/transform/transform.py

```diff
@@ -264,8 +264,13 @@ def _run(self, data_dict):
         :returns target_instances: dict - keyed by target concept -
         of lists containing dicts - representing target concept instances.
         """
-
-        target_instances = self._do_transform(data_dict)
+        if data_dict:
```

Comment:
This seems redundant with _validate_run_parameters. Could move this type of check/messaging there